### PR TITLE
Add loslesscut 1.3.0

### PR DESCRIPTION
**Pull request by andre68723**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/loslesscut.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'loslesscut' do
+  version '1.3.0'
+  sha256 'd44affab76fc4538a7968207742a2324a2cffb60b14360519650b0989b4049fb'
+
+  url "https://github.com/mifi/lossless-cut/releases/download/v#{version}/LosslessCut-darwin-x64.zip"
```
Comment (vitorgalvao) [inline Casks/loslesscut.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/loslesscut.rb (hunk 1 above)
Comment (andrev68) [inline Casks/loslesscut.rb]: thanks, done

**Comment by vitorgalvao** [comment]
Please fill the template in your top post. I’ve already done some for you.

**Comment by andrev68** [comment]
thank you again, I checked both and filled out

**Comment by vitorgalvao** [comment]
Did you run the commands? Because `brew cask style --fix` would’ve caught the trailing whitespace.

File: Casks/loslesscut.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'loslesscut' do
```
Comment (vitorgalvao) [inline Casks/loslesscut.rb]: This was wrong. Fixed in a different commit.

**Pull request merged**